### PR TITLE
Pagination of coupon list

### PR DESCRIPTION
**Pull request by roadster31**
The coupon list in the back-office is now paginated.

### CODE REVIEW

**Comment by gillesbourgeat** [comment]
:+1:

File: setup/insert.sql.tpl [hunk 1]
```diff
@@ -2006,10 +2007,11 @@ INSERT INTO `config_i18n` (`id`, `locale`, `title`, `chapo`, `description`, `pos
     (63, '{$locale}', {intl l='The minimum length required for an administrator password' locale=$locale}, NULL, NULL, NULL),
     (64, '{$locale}', {intl l='Allow an administrator to recreate a lost password (1 = yes, 0 = no)' locale=$locale}, NULL, NULL, NULL),
     (65, '{$locale}', {intl l='Send a confirmation email to newsletter subscribers (1 = yes, 0 = no)' locale=$locale}, NULL, NULL, NULL),
-    (66, '{$locale}', {intl l='Number by default of results per page for product list' locale=$locale}, NUll, NULL, NULL),
-    (67, '{$locale}', {intl l='Number by default of results per page for order list' locale=$locale}, NUll, NULL, NULL),
-    (68, '{$locale}', {intl l='Number by default of results per page for customer list' locale=$locale}, NUll, NULL, NULL),
-    (69, '{$locale}', {intl l='Enable (1) or disable (0) customer email confirmation' locale=$locale}, NULL, NULL, NULL){if ! $locale@last},{/if}
+    (66, '{$locale}', {intl l='Default number of products on product list' locale=$locale}, NUll, NULL, NULL),
```
Comment (bibich) [inline setup/insert.sql.tpl]: Instead of changing the key please change the en_US translations.

Comment (roadster31) [inline setup/insert.sql.tpl]: Done.

**Comment by bibich** [comment]
Thanks

**Pull request merged**